docs/main/troubleshooting/troubleshooting-local-storage.md

```diff
@@ -62,16 +62,15 @@ The following are different types of messages shown when the node on which local
 Warning  SelectionFailed  14m (x25 over 16m)    blockdeviceclaim-operator  no blockdevices found
 ```
 
-It means that there were no matching blockdevices after listing based on the labels. Check if there is any `block-device-tag` on the storage class and corresponding tags are available on the blockdevices also
+It means that there were no matching blockdevices after listing based on the labels. Check if there is any `block-device-tag` on the storage class and corresponding tags are available on the blockdevices.
 
 2. No devices with matching criteria
 
 ```shell hideCopy
 Warning  SelectionFailed  6m25s (x18 over 11m)  blockdeviceclaim-operator  no devices found matching the criteria
 ```
 
-It means that the there are no devices for claiming after filtering based on filesystem type and node name. Make sure the blockdevices on the node
-have the correct filesystem as mentioned in the storage class (default is `ext4`)
+It means that the there are no devices for claiming after filtering based on filesystem type and node name. Make sure the blockdevices on the node have the correct filesystem as mentioned in the storage class (default is `ext4`).
 
```

Comment:
Many contents seem to be outdated (belong to deprecated engine) in this page. You can remove them in a separate PR.